src/api/index.ts

```diff
@@ -16,16 +16,33 @@ import {
   erc20Allowances,
   FEE_TOKEN,
   exchangeOrders,
+  unregisteredTokens,
+  TOKEN_8,
 } from '../../test/data'
```

Comment:
I believe I brought it up at some point in the past.
Webpack drops unreachable code based on Env variables like `process.env.**` but only in that same statement
E.g.:
```ts
if (process.env.MOCK ==='true') //dropped

const isMock = process.env.MOCK ==='true'
if (isMock) // not dropped
```

so not removed currently

Reply:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia1.giphy.com%2Fmedia%2FFGTVmzksb2j0k%2Fgiphy.gif&token=oops" data-gitmeme-token="oops"><img src="https://media1.giphy.com/media/FGTVmzksb2j0k/giphy.gif" title="Created by gitme.me with /oops"/></a>

Reply:
Soooo, is there any action here?

Reply:
If you have it clear on how to do the ifs to instruct webpack and it obvious it good if we do it now. Otherwise, we can add a note in the issue for reviewing the bundle size.

Reply:
Yeah, I rather just leave a comment on https://github.com/gnosis/dex-react/issues/97